integrations/access/accesslist/app.go

```diff
@@ -249,10 +249,21 @@ func (a *App) getRecipientsRequiringReminders(ctx context.Context, accessList *a
 func (a *App) fetchRecipients(ctx context.Context, accessList *accesslist.AccessList, now, notificationStart time.Time) map[string]common.Recipient {
 	log := logger.Get(ctx)
 
-	allRecipients := make(map[string]common.Recipient, len(accessList.Spec.Owners))
+	var allOwners []*accesslist.Owner
+
+	allOwners, err := a.apiClient.GetAccessListOwners(ctx, accessList.GetName())
+	if err != nil {
+		// TODO(kiosion): Remove in v18; protecting against server not having `GetAccessListOwners` func.
+		log.WithError(err).Warnf("Error getting nested owners for access list %v, continuing with only explicit owners", accessList.GetName())
```

Comment:
Should this check only for NotImplemented error?

Reply:
Could do. Or do you think it'd be better to log errors here and try to continue with explicit owners only, vs. erroring on getting nested owners?

Reply:
I would treat it as a transaction - you either get all owners or none of them if there was an error. Otherwise it's harder to reason about the logic and behavior could be confusing.